client/my-sites/upgrades/checkout/style.scss

```diff
@@ -2,14 +2,9 @@
 	position: relative;
 
 	.payment-box {
-		height: 0;
 		margin-bottom: 0;
-		opacity: 0;
-		overflow: hidden;
-		padding: 0;
 		transform: translateZ(0) scale(.8); // Zoom out effect
```

Comment:
We're missing the selected class on the payment-box. (`.payment-box.selected`)This is scaling down the payment box by 20% in it's unselected state. cc @mtias were some of these other rules needed for some strange corner cases?
<img width="692" alt="screen shot 2016-08-16 at 3 30 06 pm" src="https://cloud.githubusercontent.com/assets/1270189/17718015/64ae6dd6-63c6-11e6-8138-688d544c7a35.png">